uploader/admin.py

```diff
@@ -1,3 +1,9 @@
 from django.contrib import admin
 
+from .models import FileUploadCounter
```

Comment:
Lets use the full import notation here please.

Reply:
Noted!